vizro-core/docs/pages/user-guides/custom-figures.md

```diff
@@ -1,70 +1,235 @@
 # How to create custom figures
 
-This guide explains how to create custom figures, which is useful when you need a figure that reacts to controls and actions.
+This guide explains how to create custom figures, which is useful when you need a component that reacts to
+[filter](filters.md) and [parameter](parameters.md) controls.
 
-There are two use cases where you might want to use custom figures:
+### When to use a custom figure
+You should use custom figures if **both** of the following conditions are met:
 
-* You need a figure that isn't available in our pre-defined figure functions [`vizro.figures`](../API-reference/figure-callables.md).
-* You need a figure that doesn't fit into the existing predefined components ([`Graph`][vizro.models.Graph], [`Table`][vizro.models.Table] or [`AgGrid`][vizro.models.AgGrid]).
+- You need a figure that doesn't fit into the existing predefined components ([`Graph`][vizro.models.Graph], [`Table`][vizro.models.Table] or [`AgGrid`][vizro.models.AgGrid]).
+- You need a figure that isn't available in our pre-defined figure functions [`vizro.figures`](../API-reference/figure-callables.md).
 
-Use the following approach, which is similar to creation of a [custom chart](../user-guides/custom-charts.md), do the following:
 
-1. Define a function that returns a `Dash` component.
-2. Decorate it with the `@capture("figure")` respectively.
+### Steps to create a custom figure
+
+1. Define a function that returns a [Dash component](https://dash.plotly.com/#open-source-component-libraries).
```

Comment:
```suggestion
1. Define a function that returns a [Dash component](https://dash.plotly.com/#open-source-component-libraries). This can, but does not need to, be based on code in our pre-defined figure functions in [`vizro.figures`](../API-reference/figure-callables.md).
```